helix-core/src/main/java/org/apache/helix/PropertyKey.java

```diff
@@ -627,6 +630,33 @@ public PropertyKey externalView(String resourceName) {
       return new PropertyKey(EXTERNALVIEW, ExternalView.class, _clusterName, resourceName);
     }
 
+    /**
+     * Get a property key associated with all {@link CustomizedView}
+     * @return {@link PropertyKey}
+     */
+    public PropertyKey customizedView() {
+      return new PropertyKey(CUSTOMIZEDVIEW, CustomizedView.class, _clusterName);
+    }
+
+    /**
+     * Get a property key associated with an {@link CustomizedView} of a type
+     * @param type
+     * @return {@link PropertyKey}
+     */
+    public PropertyKey customizedView(String type) {
```

Comment:
The parameter "type" is too generic. Let's make it more specific, "stateType" or even "customizedStateType". Because the type is "String" instead of a specific type, like "CustomizedStateType".  We may have different kinds of types, and by looking at "type", users do not know what it is referring to. Some existing example is "propertyType", and users know exactly what the type refers to.

Reply:
Ok. I changed it to aggregationType.